marklogic-client-api/src/main/java/com/marklogic/client/datamovement/impl/QueryBatcherImpl.java

```diff
@@ -89,6 +89,7 @@
   private JobTicket jobTicket;
   private Calendar jobStartTime;
   private Calendar jobEndTime;
+  private long maxUris;
```

Comment:
Should maxUris be initialized to Long.MAX_VALUE so that, by default, it doesn't constrain collection of results?

Alternatively, should maxUris have a Long data type and be initialized to null so that tests can ignore maxUris when it hasn't been set?